projects/ILoop/index.js

```diff
@@ -0,0 +1,15 @@
+const utils = require("../helper/utils");
+
+async function fetchTVL() {
+    const response = await utils.fetchURL("https://bot-api.prod.iloop.finance/v1/total-tvl");
```

Comment:
can we use onchain methods instead of proprietary endpoints here?

Reply:
when using onchain method I got 2 unrecognized tokens `solana:uPtSoL2qszk4SuPHNE2zqk1gDtqCq21ZE1yZCqvFTqq`
`solana:picobAEvs6w7QEknPce34wAE4gknZA9v5tTonnmHYdX`
So I decide to switch to proprietary endpoints